render_configs.sh

```diff
@@ -14,6 +16,6 @@ fi
 
 $VAULT_COMMAND -field=data -format=json "secret/dsde/firecloud/$ENV/common/firecloud-account.json" >"$INTEGRATION_OUTPUT_LOCATION/user-delegated-sa.json"
 
-if [ $ENV == perf ]; then
+if [ "$ENV" = "perf" ]; then
```

Comment:
I think the fix here is to quote ENV only. I'm pretty sure quoting perf has no effect.

Reply:
I think it's confusing to not quote perf -- it makes it look like a variable instead of a string. Many of the examples I can find online quote raw strings, including the two relevant shellcheck entries for this line: https://github.com/koalaman/shellcheck/wiki/SC3014 and https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
Probably a matter of individual taste; I'm used to not quoting but that's just me. The reason I avoid it is because bash doesn't have strings, and adding quotes to things makes it look like it does, which just furthers the confusion about bash and strings.